cinema/migrations/0002_actor_cinemahall_genre_movie_actors_movie_genres.py

```diff
@@ -0,0 +1,71 @@
+# Generated by Django 4.1 on 2024-11-29 13:32
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Actor",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("first_name", models.CharField(max_length=255)),
+                ("last_name", models.CharField(max_length=255)),
+            ],
+        ),
+        migrations.CreateModel(
+            name="CinemaHall",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("name", models.CharField(max_length=255)),
+                ("rows", models.IntegerField()),
+                ("seats_in_row", models.IntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Genre",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("name", models.CharField(max_length=50, unique=True)),
+            ],
+        ),
+        migrations.AddField(
+            model_name="movie",
+            name="actors",
+            field=models.ManyToManyField(related_name="actors", to="cinema.actor"),
```

Comment:
The `related_name` for the `actors` field is set to 'actors', which is the same as the model name. This can lead to confusion. Consider using a different name, such as 'movies' or 'movie_actors', to avoid conflicts and improve clarity.